libcontainer/configs/cgroup.go

```diff
@@ -57,6 +57,11 @@ type Cgroup struct {
 	// MEM to use
 	CpusetMems string `json:"cpuset_mems"`
 
+	// Process limit; set to `0' to disable limit.
+	// While technically `0' is a valid PID limit, it does not make sense in the
+	// context of a container -- it is identical to a limit of `1'.
```

Comment:
Gah, that should read `<= 0`. Whoops. I'll send a PR to fix the comment when I get a chance. The problem with `0` being a valid limit is that it makes the spec non-backwards compatible (and you have to specify it in `config.json`) ... unless the change in #369 changes the default?